deepmd/infer/deep_density.py

```
@@ -0,0 +1,121 @@
+        -------
+        density
+            The density on the grid points, in shape (nframes, ngrid).
+        """
+        (
+            coords,
+            cells,
+            atom_types,
+            fparam,
+            aparam,
+            nframes,
+            natoms,
```

Ruff (0.16.2)

[warning] Unpacked variable `natoms` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)